code/modules/mob/living/simple_animal/abnormality/waw/naked_nest.dm

```diff
@@ -0,0 +1,360 @@
+	..()
+	if(origin_nest)
+		for(var/mob/living/simple_animal/hostile/abnormality/naked_nest/N in oview(vision_range, src))
+			if(origin_nest == N.tag)
+				Goto(N, 5, 0)
+
+/mob/living/simple_animal/hostile/naked_nested
+	name = "naked nested"
+	desc = "A humanoid form covered in scales with numerous holes. It looks like it has reinforced itself with its hosts armor."
+	icon = 'ModularTegustation/Teguicons/tegumobs.dmi'
+	icon_state = "nakednest_minion"
+	icon_living = "nakednest_minion"
+	icon_dead = "nakednest_miniondead"
+	faction = list("Naked_Nest")
+	deathmessage = "collapses into a unrecognizable pile of scales, shredded clothing, and broken serpents."
+	melee_damage_lower = 10
+	melee_damage_upper = 30
+	melee_damage_type = RED_DAMAGE
+	armortype = RED_DAMAGE
+	maxHealth = 300
+	health = 300
+	stat_attack = CONSCIOUS //When you are put into crit the nested will continue to transform into a nest. I thought about having the nested infest you if your in crit but that seemed a bit too cruel.
+	damage_coeff = list(BRUTE = 1, RED_DAMAGE = 0.6, WHITE_DAMAGE = 0.8, BLACK_DAMAGE = 1.2, PALE_DAMAGE = 1.5)
+	mob_size = MOB_SIZE_HUMAN
+	minbodytemp = INHOSPITABLE_FOR_NESTING
+	var/nestingtimer
+
+/mob/living/simple_animal/hostile/naked_nested/Initialize()
+	. = ..()
+	nestingtimer = world.time + (40 SECONDS)
+
+/mob/living/simple_animal/hostile/naked_nested/Life()
+	. = ..()
+	if(stat == DEAD && buffed == 0)
+		buffed = 1
+		nestingtimer = nestingtimer + (120 SECONDS)
+	if(nestingtimer <= world.time && !target)
+		new /mob/living/simple_animal/hostile/abnormality/naked_nest(get_turf(src))
+		playsound(get_turf(src), 'sound/misc/moist_impact.ogg', 10, 1)
+		qdel(src)
+
+	//Status Effect
+/datum/status_effect/serpents_host // its final destination is your frontal lobe
+	id = "serpents_host"
+	status_type = STATUS_EFFECT_UNIQUE
+	duration = 2400 //4 minutes
+	alert_type = null
+	var/cured = 0
+	var/physical_symptoms
+	var/presented_symptoms
+	var/originalskintone
+	var/extra_time = 0.8 SECONDS //doubles remaining time
+
+/datum/status_effect/serpents_host/on_apply()
+	. = ..()
+	if(ishuman(owner))
+		owner.add_movespeed_mod_immunities(type, /datum/movespeed_modifier/damage_slowdown)
+		owner.add_movespeed_mod_immunities(type, /datum/movespeed_modifier/justice_attribute) //will test to see if can be cured.
+		owner.faction += "Naked_Nest"
+		var/mob/living/carbon/human/H = owner
+		originalskintone = H.skin_tone
+	physical_symptoms = world.time + (180 SECONDS)
+
+/datum/status_effect/serpents_host/tick()
+	. = ..()
+	if(ishuman(owner))
+		var/mob/living/carbon/human/H = owner
+		H.adjustSanityLoss(-0.1) //the serpents final destination is your frontal lobe
+		H.adjustOrganLoss(ORGAN_SLOT_BRAIN, 0.1)
+		if(H.bodytemperature <= INHOSPITABLE_FOR_NESTING || H.reagents.has_reagent(/datum/reagent/consumable/wellcheers_purple, 20) || H.reagents.has_reagent(/datum/reagent/consumable/red_queen, 20)) //cure conditions
```

Comment:
This is two wellcheers or a cup of inaccessible tea

Reply:
Changed it to just temperature.